### PR TITLE
Deployment preview documentation

### PR DESCRIPTION
**Pull request by Anshuman71**
## What changed?

Explain what's a deployment preview, how it helps the user, and how they can add one.

**Preview**:
- https://appkit-pr-2964.commercetools.vercel.app/custom-applications/help-needed#using-a-test-or-staging-environment-for-my-custom-application
- https://appkit-pr-2964.commercetools.vercel.app/custom-applications/concepts/deployment-previews

### CODE REVIEW

File: website/src/content/api-reference/deployment-previews.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+2. Deploying the Custom Application preview to your hosting provider.
+3. [Adding the deployment preview](https://docs.commercetools.com/merchant-center/managing-custom-applications#adding-deployment-previews) to your Custom Application settings.
+
+# Building deployment previews
+
+Each deployment preview has its unique and stable deployment URL. To build a deployment preview, you must set the `env.production.url` value in the `custom-application-config.json` file to the deployed application. You can follow one of the below approaches:
+
+## Using environment variables
+
+To configure your application URL, you can use environment variables. Then, you can assign different values to the specified environment variable in each deployment environment. The Custom Application `build` script will automatically resolve the application URL from the environment variables.
```
Comment (kark) [inline website/src/content/api-reference/deployment-previews.mdx:25]: nit: maybe [`mc-scripts build`](https://docs.commercetools.com/custom-applications/api-reference/cli#build), not the Custom Application `build` script

**Review by kark — COMMENTED** [review]
Thank you for the write-up ❤️
It seems technically correct to me.

**Comment by Anshuman71** [comment]
@mmaltsev-ct, I’ve added a basic release note. Feel free to suggest changes.

File: website/src/content/api-reference/deployment-previews.mdx (hunk 1 above)
Comment (Anshuman71) [reply website/src/content/api-reference/deployment-previews.mdx]: Thanks. Good suggestion.

File: website/src/data/navigation.yaml [hunk 2]
```diff
@@ -76,6 +76,8 @@
       path: /api-reference/create-mc-app
     - title: Custom Webpack Config
       path: /api-reference/custom-webpack-config
+    - title: Deployment previews
+      path: /api-reference/deployment-previews
```
Comment (emmenko) [inline website/src/data/navigation.yaml:80]: Hmm I'm not sure this should go into the API Reference, maybe into the Concepts section.

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+date: 2023-02-24
+title: Custom Application deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now lets you add deployment previews, where you can test new features before publishing your Custom Application to the production environment.
```
Comment (emmenko) [inline website/src/releases/2023-02/deployment-previews.mdx:11]: Please add a link to the docs where we explain what the feature is and how to use it.

File: website/src/content/help-needed.mdx [hunk 4]
```diff
@@ -49,82 +49,7 @@ If you are interested in more advanced functionalities, let us know and open a [
...
-import { entryPointUriPathToPermissionKeys } from '@commercetools-frontend/application-shell/ssr';
-
-export const entryPointUriPath = typeof window === 'undefined'
-  ? process.env.ENTRY_POINT_URI_PATH
-  : window.app.entryPointUriPath;
-
-export const PERMISSIONS = entryPointUriPathToPermissionKeys(entryPointUriPath);
-```
-</Info>
+You can set up a test or staging environment for your Custom Application by adding a [deployment preview](./api-reference/deployment-previews).
```
Comment (emmenko) [inline website/src/content/help-needed.mdx:52]: Nit: Should we maybe leave the old docs and add an info message explaining there is now a better way of doing this?
At least for the time being, then we can remove the old content.

File: website/src/releases/2023-02/deployment-previews.mdx (hunk 3 above)
Comment (emmenko) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Also, it might be good to have some sort of migration steps described.

File: website/src/content/api-reference/deployment-previews.mdx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+```json
+{
+  // ...more configurations
+  "env": {
+    "production": {
+      "url": "${env:URL}",
+    },
+  // ...more configurations
+}
+```
```
Comment (emmenko) [inline website/src/content/api-reference/deployment-previews.mdx:44]: This is messing up the syntax highlighting, as comments are not supported in JSON. Please remove them.

````suggestion
```json title="custom-application-config.json"
{
  "env": {
    "production": {
      "url": "${env:APPLICATION_URL}",
    }
  }
}
```
````

File: website/src/content/api-reference/deployment-previews.mdx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
+
+</Subtitle>
+
+This document describes the necessary steps to build and add deployment previews to your Custom Application in the Merchant Center.
```
Comment (emmenko) [inline website/src/content/api-reference/deployment-previews.mdx:11]: Nit: the steps to add deployment previews belong more to the MC docs. I would focus here only on the application config setup.

File: website/src/content/api-reference/deployment-previews.mdx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+The process involves:
+
+1. Building deployment previews for your Custom Application.
+2. Deploying the Custom Application preview to your hosting provider.
+3. Adding the deployment preview to the Custom Application settings.
+
+# Building deployment previews
+
+Each deployment preview has its unique and stable deployment URL. To build a deployment preview, you must set the `env.production.url` value in the `custom-application-config.json` file to the deployed application. You can follow one of the below approaches:
```
Comment (emmenko) [inline website/src/content/api-reference/deployment-previews.mdx:21]: Nit: is `stable` the right term here? Feels a bit weird to me...maybe `static`?

Comment (emmenko) [inline website/src/content/api-reference/deployment-previews.mdx:21]: I would like to also explain WHY the user has to configure the URL, the context is missing a bit here.

Comment (emmenko) [inline website/src/content/api-reference/deployment-previews.mdx:21]: > You can follow one of the below approaches:

Aren't both steps required? Maybe this section should be reworked.

File: website/src/content/api-reference/deployment-previews.mdx [hunk 8]
```diff
@@ -0,0 +1,67 @@
+---
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
```
Comment (emmenko) [inline website/src/content/api-reference/deployment-previews.mdx:7]: Nit: I would rephrase this a bit, it might seem 

```suggestion
Deployment previews allow you to preview changes to your Custom Application using a unique URL.
```

**Review by emmenko — COMMENTED** [review]
Thanks for starting to work on this.

I left some comments here and in the MC docs. Please consider both as the overall context.

File: website/src/content/api-reference/deployment-previews.mdx (hunk 7 above)
Comment (Anshuman71) [reply website/src/content/api-reference/deployment-previews.mdx:21]: Nope. But you can use them in conjunction. 

The ENV approach isn't dependent on Git, you can have different env variables on your hosting provider, and it'll work perfectly. 

Same goes for the Git approach, if you hardcode the values, you need to keep them separate with branching. In this case, the hosting provider relies on the value present in the build branch (configured on the hosting platform side)

File: website/src/content/api-reference/deployment-previews.mdx (hunk 8 above)
Comment (Anshuman71) [reply website/src/content/api-reference/deployment-previews.mdx:7]: Thanks, @emmenko. It looks clearer. Although, "allows" is a prohibited word in our styleguide. I'll rephrase to keep your intent.

20-02-2023-at-15-21-20@2x

File: website/src/content/api-reference/deployment-previews.mdx (hunk 7 above)
Comment (Anshuman71) [reply website/src/content/api-reference/deployment-previews.mdx:21]: Sure.

File: website/src/content/api-reference/deployment-previews.mdx (hunk 6 above)
Comment (Anshuman71) [reply website/src/content/api-reference/deployment-previews.mdx:11]: That's true. But, to complete the loop, we have mentioned the link to MC docs as the last step of this process.

File: website/src/content/help-needed.mdx (hunk 4 above)
Comment (Anshuman71) [reply website/src/content/help-needed.mdx:52]: Keeping the old content will be contradictory. But I'd like to understand the "At least for the time being" part to make a better point.

File: website/src/data/navigation.yaml (hunk 2 above)
Comment (Anshuman71) [reply website/src/data/navigation.yaml:80]: Sure. I picked API Reference from the docs link in the feature preview on Staging Merchant Center.

File: website/src/releases/2023-02/deployment-previews.mdx (hunk 3 above)
Comment (Anshuman71) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Sure. The release note is in WIP. @mmaltsev-ct will help me.
- I'll add the link to docs.
- Can you highlight a bit more on the migration steps required?

Comment (mmaltsev-ct) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Here is my suggestion for the release notes:

Comment (mmaltsev-ct) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Custom Applications now let you generate deployment previews, where you can test new features before publishing your application to the production environment. With the first version of this feature, it is possible to manually create static environments using the Merchant Center. A static URL, where the Custom Application will be deployed, can then be assigned to each of the environments.

You can read more about this feature in the new [Deployment previews](https://docs.commercetools.com/custom-applications/concepts/deployment-previews) chapter of our documentation.

As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).

Comment (mmaltsev-ct) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: I think the migration steps should be better described in the docs themselves rather than in the release notes.

Comment (Anshuman71) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Thanks, @mmaltsev-ct. I'll adapt your suggestion in the release note. Will you also review the release note on the MC PR https://github.com/commercetools/commercetools-docs/pull/2900?

File: website/src/content/api-reference/deployment-previews.mdx (hunk 7 above)
Comment (Anshuman71) [reply website/src/content/api-reference/deployment-previews.mdx]: @emmenko I’ve removed this section to avoid confusion, as we discussed.

File: website/src/content/help-needed.mdx (hunk 4 above)
Comment (Anshuman71) [reply website/src/content/help-needed.mdx]: @emmenko while changing this document, I reliazied that we should keep the older docs for teams that are using the workaround. Hence, I've restructured the section to keep both but labeling old approach as "workaround" and mentioning deployment previews at top.

**Review by emmenko — COMMENTED** [review]
Awesome work, all good from my side now!

**Review by mmaltsev-ct — APPROVED** [review]
Looks great, thank you @Anshuman71!

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 9]
```diff
@@ -0,0 +1,15 @@
+---
+date: 2023-02-24
+title: Custom Application deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
```
[the remark this answers is not in the document]
Comment (emmenko) [reply website/src/releases/2023-02/deployment-previews.mdx:7]: @melindafekete I wanted to sync with you actually about the topics and see if there is a list of commonly used ones. I feel like often times we just pick some random words...

[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/releases/2023-02/deployment-previews.mdx:7]: Oh, interesting, I can see that we don't have a set list defined for Custom Apps (like we do for the HTTP API, MC, Studio, etc.). I will create a Google doc for us to brainstorm and define which set of topics would be the most appropriate.

[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/releases/2023-02/deployment-previews.mdx:7]: Current examples for other release notes are here: https://commercetools.atlassian.net/wiki/spaces/DOCS/pages/105818548/Writing+release+notes

[the remark this answers is not in the document]
Comment (emmenko) [reply website/src/releases/2023-02/deployment-previews.mdx:7]: Sounds good, let's continue in the google doc. Thanks!

[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/releases/2023-02/deployment-previews.mdx:7]: https://docs.google.com/document/d/12fMDajZPyrTr31S34E-TKsdOcTJOEpY214DSN_kRXgk/edit#

File: website/src/content/concepts/deployment-previews.mdx [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you preview your Custom Application changes using a unique URL.
+
+</Subtitle>
+
+To test your Custom Application features before releasing them to the production environment, you can add a deployment preview. For example, you can add a deployment preview to try all the updates in the `staging` branch and another to test some experimental features.
```
[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/content/concepts/deployment-previews.mdx:11]: I think it was for the updates to the custom app but I'll rephrase to something clear.

File: website/src/content/concepts/deployment-previews.mdx [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  }
+}
+```
+
+With the above setup, you can build deployment previews based on the `APP_URL` environment variable value. How you set the environment is up to you. For example, you could set the `APP_URL` to different values in different hosting environments or create a `.env` file in your project and commit different values for the `APP_URL` variable in different branches.
+
+# Deploying the Custom Application preview
+
+Once you've built your Custom Application using the correct `env.production.url` value. You can deploy the Custom Application preview the same as you deploy your Custom Application.
```
[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/content/concepts/deployment-previews.mdx]: Can you recheck this one? I think it's deployed already.

File: website/src/content/concepts/deployment-previews.mdx [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+The only requirement is that the deployed application URL (`APP_URL`) is static, so you can specify it on the Merchant Center while [Adding the deployment preview](https://docs.commercetools.com/merchant-center/managing-custom-applications#adding-deployment-previews).
```
[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/content/concepts/deployment-previews.mdx]: Can you recheck this one? I think it's deployed already.

File: website/src/content/concepts/deployment-previews.mdx [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+
+See [Deployment Examples](./deployment-examples) to get up and running with some of the common cloud hosting providers.
+
+<Info>
+
+You must set the `APP_URL` environment variable on your hosting provider.
```
[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/content/concepts/deployment-previews.mdx:45]: Can you recheck this one? I think it's deployed already.

File: website/src/content/concepts/deployment-previews.mdx [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+
+You must host the deployment previews separately from one another and assign a unique but static URL. You can add as many deployment previews as you like. In Merchant Center, the deployment previews are available on a unique URL, different than the production environment.
+
+This document describes the necessary steps to build and add deployment previews to your Custom Application. The process involves:
+
+1. Building deployment previews for your Custom Application.
+2. Deploying the Custom Application preview to your hosting provider.
+3. Adding the deployment preview to the Custom Application settings.
+
+# Building deployment previews
```
[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/content/concepts/deployment-previews.mdx]: Can you recheck this one? I think it's deployed already.

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+  - Configuration
+---
+
+Custom Applications now let you generate [Deployment previews](./concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
+
+You can add a deployment preview using the Merchant Center by assigning it a static URL of the deployed Custom Application.
```
[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/releases/2023-02/deployment-previews.mdx]: @Anshuman71 let me know if you have any questions about this one :)

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+date: 2023-02-24
+title: Custom Application deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [Deployment previews](./concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
```
[the remark this answers is not in the document]
Comment (mmaltsev-ct) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: @melindafekete, no, this was not possible before :) The legacy approach is just a workaround. We need to highlight this then if it's not clear. Regarding the timeline: we have a more advanced vision for this feature that we would like to implement later in the future. I'd suggest coming back to the discussion around removing the legacy instructions when we will be releasing the next iteration. Here are the updated release notes, please let me know what you think (cc: @Anshuman71):

Custom Applications now let you generate [Deployment previews](https://github.com/commercetools/merchant-center-application-kit/blob/docs/deployment-previews/website/src/releases/2023-02/concepts/deployment-previews), where you can test new features before publishing your application to the production environment.

The [legacy workaround](https://docs.commercetools.com/custom-applications/help-needed#using-a-test-or-staging-environment-for-my-custom-application) required the creation of multiple Custom Applications and the usage of a dynamic configuration with the help of environment variables. We implemented the new feature to decrease the maintenance efforts for the engineering teams working on Custom Applications and to improve the developer UX in general. Please beware that as we continue improving this feature, we will eventually remove the instructions for the legacy solution from our documentation.

As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).

[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Thanks, @mmaltsev-ct. @melindafekete and @mmaltsev-ct, I've reworked the suggestion. I don't think sharing the "legacy method docs deprecation" is a good idea without the fixed date. Maybe we can share the date in a future release note of deployment previews.

---

Custom Applications now let you generate [Deployment previews](../concepts/deployment-previews), where you can test new features before publishing your application to the production environment.

Emulating Custom Application previews with the [legacy workaround](../help-needed#legacy-workaround) required creating multiple Custom Applications and dynamic configurations using environment variables. Deployment previews provide a better developer experience by complementing your existing Custom Application. 

We recommend using deployment previews for the upcoming releases of your Custom Applications, but we'll keep the legacy workaround documentation in the meantime.

As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).

[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Agree with you @Anshuman71 that we shouldn't mention the legacy docs deprecation without a timeline, but then let's not mention the legacy docs at all. Let's just make the recommendation to use the new feature and stop there. We can always make the announcement later if we have the specifics.

As for the rest of the release note, it looks good, I'm just not sure what you mean by 'by complementing your existing Custom Application'? I also liked Maxim's point about the lower maintenance efforts, which seems to have not made it into your version.

File: website/src/content/help-needed.mdx [hunk 17]
```diff
@@ -49,13 +49,19 @@ If you are interested in more advanced functionalities, let us know and open a [
 
 # Using a test or staging environment for my Custom Application
 
-During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment.
+During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment. This can be accomplished using [deployment previews](./concepts/deployment-previews) which allow you to inspect different versions of a Custom Application.
```
Comment (emmenko) [inline website/src/content/help-needed.mdx:52]:
```suggestion
During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment. This can be accomplished using [deployment previews](/concepts/deployment-previews) which allow you to inspect different versions of a Custom Application.
```

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 18]
```diff
@@ -0,0 +1,17 @@
...
+date: 2023-03-03
+title: Custom Applications deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [Deployment previews](../concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
```
Comment (emmenko) [inline website/src/releases/2023-02/deployment-previews.mdx:11]:
```suggestion
Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features before publishing your application to the production environment.
```

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 19]
```diff
@@ -0,0 +1,17 @@
...
+
+Emulating Custom Application previews with the [legacy workaround](../help-needed#legacy-workaround) required you to create multiple Custom Applications and dynamic configurations using environment variables. Deployment previews work without creating a new Custom Application in the Merchant Center, leading to less maintenance and better developer experience.
```
Comment (emmenko) [inline website/src/releases/2023-02/deployment-previews.mdx:13]:
```suggestion
Emulating Custom Application previews with the [legacy workaround](/help-needed#legacy-workaround) required you to create multiple Custom Applications and dynamic configurations using environment variables. Deployment previews work without creating a new Custom Application in the Merchant Center, leading to less maintenance and better developer experience.
```

File: website/src/releases/2023-02/deployment-previews.mdx (hunk 18 above)
Comment (emmenko) [reply website/src/releases/2023-02/deployment-previews.mdx:11]: Nit: the sentence might be a bit confusing, as it seems that you can't use the production version without a preview.

```suggestion
Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features without affecting the production version of your application.
```

File: website/src/releases/2023-02/deployment-previews.mdx (hunk 19 above)
Comment (emmenko) [reply website/src/releases/2023-02/deployment-previews.mdx:13]: Nit: the "dynamic configurations using environment variables" is still needed for the URL part, I would maybe rephrase this a bit.

```suggestion
If you were using the [legacy workaround](../help-needed#legacy-workaround) by creating multiple Custom Applications with different entry points, we recommend migrating  to deployment previews where you only need to configure one Custom Application. This aims to provide less maintenance and a better developer experience.
```

File: website/src/releases/2023-02/deployment-previews.mdx [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+
+We recommend using deployment previews for the upcoming releases of your Custom Applications.
```
Comment (emmenko) [inline website/src/releases/2023-02/deployment-previews.mdx:15]: Can be removed

```suggestion
```

**Review by emmenko — COMMENTED** [review]
Did a final review. I would rephrase a couple of things but otherwise looks good! 🤗

Thanks a lot for the effort and help so far ❤️

File: website/src/content/concepts/deployment-previews.mdx [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+
+Once you've built your Custom Application using the correct `env.production.url` value. You can deploy the Custom Application preview the same way you deploy your Custom Application. The only requirement is that the deployed application URL (`APP_URL`) is static, so you can specify it in the Merchant Center when [adding the deployment preview](https://docs.commercetools.com/merchant-center/managing-custom-applications#adding-deployment-previews).
+
+<Info>
+
+You must set the `APP_URL` environment variable on your hosting provider.
+
+</Info>
+
+See [Deployment Examples](./deployment-examples) to get up and running with some of the common cloud hosting providers.
```
[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/content/concepts/deployment-previews.mdx:49]: Why lowercase 'd' but uppercase 'E'? I think @Anshuman71 matched the casing of the original title there, but we can also go for something like 'Use our deployment examples to get up and running with..'

[the remark this answers is not in the document]
Comment (emmenko) [reply website/src/content/concepts/deployment-previews.mdx:49]: Sorry my bad here. However, I noticed we're not using the casing consistently. What it the rule to follow here?

File: website/src/releases/2023-02/deployment-previews.mdx (hunk 15 above)
[the remark this answers is not in the document]
Comment (Anshuman71) [reply website/src/releases/2023-02/deployment-previews.mdx]: @melindafekete, this section was reworked heavily let me know if it still has ambiguity.

File: website/src/content/concepts/deployment-previews.mdx (hunk 21 above)
[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/content/concepts/deployment-previews.mdx:49]: We try to follow the same casing and wording as the original source, but sometimes you can end up with really funny sentences that way. If you're changing the wording to match the sentence, then it should be all lowercase (or the casing that would apply without the link).

[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/content/concepts/deployment-previews.mdx:49]: The problem here is that the casing of headings is overall inconsistent within the Custom Apps docs. For example: 'Getting Started', but 'Maintenance guidelines', 'Deployment Examples' but 'Deploying to Vercel'. The first step would be to fix this and make all headings sentence case. Then we can also fix the casing of linked text.

[the remark this answers is not in the document]
Comment (melindafekete) [reply website/src/content/concepts/deployment-previews.mdx:49]: @Anshuman71 could you add this as a task to the Improve Custom Apps docs project? 🙏

[the remark this answers is not in the document]
Comment (emmenko) [reply website/src/content/concepts/deployment-previews.mdx:49]: Ok thanks, let's take this as a follow up then.

File: website/src/content/concepts/deployment-previews.mdx [hunk 22]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (emmenko) [inline website/src/content/concepts/deployment-previews.mdx:49]:
```suggestion
See [Deployment Examples](/deployment-examples) to get up and running with some of the common cloud hosting providers.
```

File: website/src/releases/2023-03/deployment-previews.mdx [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features without affecting the production version of your application.
+
+If you're using the [legacy workaround](../help-needed#legacy-workaround) by creating multiple Custom Applications with different entry points, we recommend migrating to deployment previews for a better developer experience with less maintenance, where you only need to configure one Custom Application.
```
Comment (emmenko) [inline website/src/releases/2023-03/deployment-previews.mdx:13]:
```suggestion
If you're using the [legacy workaround](/help-needed#legacy-workaround) by creating multiple Custom Applications with different entry points, we recommend migrating to deployment previews for a better developer experience with less maintenance, where you only need to configure one Custom Application.
```

**Review by emmenko — APPROVED** [review]
🥳

File: website/src/releases/2023-03/deployment-previews.mdx [hunk 24]
```diff
@@ -0,0 +1,15 @@
+---
+date: 2023-03-03
```
Comment (melindafekete) [inline website/src/releases/2023-03/deployment-previews.mdx:2]: Just updating ahead of the release for Monday:
```suggestion
date: 2023-03-06
```

File: website/src/content/concepts/deployment-previews.mdx [hunk 25]
```diff
@@ -0,0 +1,55 @@
...
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you preview your Custom Application changes using a unique URL.
+
+</Subtitle>
+
+To test your Custom Application features before releasing them to the production environment, you can add a deployment preview. For example, you can add a deployment preview to try new features in your Custom Applcation's `staging` branch.
```
Comment (melindafekete) [inline website/src/content/concepts/deployment-previews.mdx:11]:
```suggestion
To test your Custom Application features before releasing them to the production environment, you can add a deployment preview. For example, you can add a deployment preview to try new features in your Custom Application's `staging` branch.
```

File: website/src/releases/2023-03/deployment-previews.mdx [hunk 26]
```diff
@@ -0,0 +1,15 @@
...
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now let you generate [deployment previews](/concepts/deployment-previews), where you can test new features without affecting the production version of your application.
+
+If you're using the [legacy workaround](/help-needed#legacy-workaround) by creating multiple Custom Applications with different entry points, we recommend migrating to deployment previews for a better developer experience with less maintenance, where you only need to configure one Custom Application.
```
Comment (melindafekete) [inline website/src/releases/2023-03/deployment-previews.mdx:13]:
```suggestion
If you're using the [legacy workaround](/help-needed#legacy-workaround) of creating multiple Custom Applications with different entry points, we recommend migrating to deployment previews. They provide a better developer experience with less maintenance since you only need to configure one Custom Application.
```

**Review by melindafekete — APPROVED** [review]
Approved - pending the typo fix :)

File: website/src/content/concepts/deployment-previews.mdx (hunk 25 above)
Comment (Anshuman71) [reply website/src/content/concepts/deployment-previews.mdx:11]: Good catch

**Comment by Anshuman71** [comment]
@melindafekete and @mmaltsev-ct, thanks for supporting this PR. I've applied all the latest changes, and the PR is ready for merging.

- [ ] Double-check the release date before merging.

File: website/src/releases/2023-03/deployment-previews.mdx (hunk 26 above)
Comment (emmenko) [reply website/src/releases/2023-03/deployment-previews.mdx:13]: @Anshuman71 What about this suggestion? ⬆️

File: website/src/releases/2023-03/deployment-previews.mdx (hunk 24 above)
Comment (emmenko) [reply website/src/releases/2023-03/deployment-previews.mdx:2]: @Anshuman71 This needs to be updated as well.

File: website/src/releases/2023-03/deployment-previews.mdx [hunk 27]
```diff
@@ -0,0 +1,15 @@
+---
+date: 2023-03-03
+title: Custom Applications deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
```
Comment (emmenko) [inline website/src/releases/2023-03/deployment-previews.mdx:5]:
```suggestion
type: feature
```

**Review by emmenko — APPROVED** [review]
